src/validations/index.ts

```diff
@@ -0,0 +1,3 @@
+import isValidPublicKey from "./isValidPublicKey";
+
+export { isValidPublicKey };
```

Comment:
Do we really need this file?

Reply:
solved in commit dbec14a066bcf10a582a69fde564beb4d9fec934.